### ISSUE TITLE
Question: can we remove catpandoc to reduce bundle size?

### ISSUE THREAD

**Issue by milahu** [FHPythonUtils/Cli2Gui#5]
on nix, `pypandoc` pulls in these dependencies:

```
these 4 paths will be fetched (20.74 MiB download, 164.43 MiB unpacked):
  /nix/store/5gm4rzsdd40i9ph97bxs5bwrfanmg11v-lua-5.3.6
  /nix/store/7iyh4zqkmk8i6bpkdg99zrqhbckmy685-python3.9-pypandoc-1.7.2
  /nix/store/sf07y0qw3bn11l7zj71kpadf65mffipb-pandoc-2.14.0.3-data
  /nix/store/yvbr6knw00jl6wd7afcc8f9knl60h3hb-pandoc-2.14.0.3
```

feels a bit heavy. do we really need pandoc?

**Comment by FredHappyface** [comment]
It can be reasonably handy when it comes to parsing readmes and similar for a menu. But I agree that it should be optional here to slim the library a little

**Comment by FredHappyface** [comment]
Fixed in 2022.1 let me know if that has solved things for you

**Comment by milahu** [comment]
yepp. thanks : )
